vendor/engines/sanger_sequencing/db/migrate/20250120141340_add_sample_primer_name.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class AddSamplePrimerName < ActiveRecord::Migration[4.2]
+  def up
+    unless ActiveRecord::Base.connection.column_exists?(:sanger_sequencing_samples, :primer_name)
```

Comment:
Why do we need this check in order to fix Oracle? How did it break that DB before?

Reply:
Initially I was using the argument `if_not_exsits: true` so it does not raise an error if this column exists already (because it was added by the acgt module). When I tried this on NU it raised the error anyway, my guess is that the oracle adapter doesn't handle this flag, but didn't research much about it and I found no related issues